app/models/wpcc/calculator_delegator.rb

```diff
@@ -0,0 +1,34 @@
+require 'active_model'
```

Comment:
Yes, it is needed to require, but it is already required on the main file: https://github.com/moneyadviceservice/wpcc/blob/master/lib/wpcc.rb#L1
So we can remove this line above.